package.json

```diff
@@ -11,7 +11,7 @@
     "prettier": "prettier --write .",
     "publish-npm": "lerna publish",
     "rollup": "lerna run rollup",
-    "doc": "lerna run doc",
+    "doc": "node doc/update.js",
```

Comment:
This is slick having the documentation done using node...I guess there really isn't any other way since it needs to run notebooks, but it still seems neat that this isn't a .sh file or something.